oidc_cli/storage/storage.go

```diff
@@ -20,3 +23,31 @@ type Storage interface {
 
 	MarshalOpts() []client.MarshalOpts
 }
+
+func GetStorage(clientID string, issuerURL string) (Storage, error) {
```

Comment:
might be able to rename this to storage.Get since now in a storage package

Reply:
or GetOIDC() --- since specific to OIDC(bc parameterized by client_id, issuerURL)